client-js/app/elements/labrad-manager.ts

```diff
@@ -1,9 +1,14 @@
-import {ConnectionInfo} from '../scripts/manager';
+import {ConnectionInfo, ManagerApi} from '../scripts/manager';
 
 @component('labrad-manager')
 export class LabradManager extends polymer.Base {
 
   @property({type: Array, notify: true})
   connections: Array<ConnectionInfo>;
 
+  mgr: ManagerApi;
+
+  closeConnection(event) {
+    this.mgr.connectionClose(event.currentTarget.connId);
```

Comment:
I did wonder that, I tried to make the actual buttons tiny to avoid it. 
`window.confirm` is a big gross, I'll do it properly with a dialog and resubmit.